src/components/layout/sidebar/index.tsx

```diff
@@ -17,6 +17,88 @@ const Sidebar = () => {
   const { isSidebarOpen, sidebarOpenToggle } = useLayoutContext()
   const breakpoint = useBreakpoint()
   const sidebarRef = useRef(null)
+  const generalComponent = [
+    {
+      href: '/component/alert',
+      text: 'Alert',
+    },
+    {
+      href: '/component/badge',
+      text: 'Badge',
+    },
+    {
+      href: '/component/breadcrumb',
+      text: 'Breadcrumb',
+    },
+    {
+      href: '/component/buttons',
+      text: 'Buttons',
+    },
+    {
+      href: '/component/card',
+      text: 'Card',
+    },
+    {
+      href: '/component/carousel',
+      text: 'Carousel',
+    },
+    {
+      href: '/component/collapse',
+      text: 'Collapse',
+    },
+    {
+      href: '/component/dropdown',
+      text: 'Dropdown',
+    },
+    {
+      href: '/component/form',
+      text: 'Form',
+    },
+    {
+      href: '/component/list-group',
+      text: 'List Group',
+    },
+    {
+      href: '/component/media-object',
+      text: 'Media Object',
+    },
+    {
+      href: '/component/modal',
+      text: 'Modal',
+    },
+    {
+      href: '/component/nav',
+      text: 'Nav',
+    },
+    {
+      href: '/component/navbar',
+      text: 'Navbar',
+    },
+    {
+      href: '/component/pagination',
+      text: 'Pagination',
+    },
+    {
+      href: '/component/popover',
+      text: 'Popover',
+    },
+    {
+      href: '/component/progress',
+      text: 'Progress',
+    },
+    {
+      href: '/component/table',
+      text: 'Table',
+    },
+    {
+      href: '/component/tooltip',
+      text: 'Tooltip',
+    },
+    {
+      href: '/component/typography',
+      text: 'Typography',
+    },
+  ]
```

Comment:
Please rewrite the `component` route to use the plural: `components`; e.g. `/component/table`.